OPHD/ShellOpenPath.cpp

```diff
@@ -8,7 +8,8 @@
 namespace
 {
 	#if defined(_WIN32)
-		constexpr const std::string_view ShellOpenCommand{"start"};
+		#include <Windows.h>
+		#include <shellapi.h>
```

Comment:
> OPHD/ShellOpenPath.cpp:11:12: fatal error: Windows.h: No such file or directory
   11 |   #include <Windows.h>

I assume it's because the filesystem is case sensitive on Linux. Maybe try a lowercase `w`.